src/icons/Icon.js

```diff
@@ -47,9 +48,9 @@ const Icon = props => {
         style={StyleSheet.flatten([
           (reverse || raised) && styles.button,
           (reverse || raised) && {
-            borderRadius: size + 4,
-            height: size * 2 + 4,
-            width: size * 2 + 4,
+            borderRadius: (size * reverseRatio) / 2,
+            height: size * reverseRatio,
+            width: size * reverseRatio,
```

Comment:
This would be a breaking change no?

Say example size = 30

Before: 
```
height: 30 * 2 + 4 = 64
```

After:
```
height: 30 * 1.5 = 45
```

This would result in different sized icons if you were using icon previously

Reply:
We can define `reverseRatio` to 2 by default, to not change anything.
I removed the `+ 4`, I don't see why it is there

Reply:
Not sure why it's there either, but it means we'd have to update all the components that use Icon (which is a lot). As well as in user's apps. Your call

Reply:
We can see that later
For now, let's leave `+ 4` and reverseRatio to `2` by default to not change anything